tests/mongo-go-driver.yml

```diff
@@ -166,1106 +162,95 @@ results:
   
   mongodb:
     stats:
-      fail: 30
-      skip: 276
-      pass: 2733
+      fail: 31
+      skip: 273
+      pass: 2584
     ignore:
-      - go.mongodb.org/mongo-driver/mongo/integration/TestUnifiedSpecs
-      - go.mongodb.org/mongo-driver/mongo/integration/TestClient_BSONOptions
-      - go.mongodb.org/mongo-driver/mongo/integration/TestCommandLoggingAndMonitoringProse
-      - go.mongodb.org/mongo-driver/mongo/integration/TestSearchIndexProse
+      - go.mongodb.org/mongo-driver/internal/integration/TestUnifiedSpecs
+      - go.mongodb.org/mongo-driver/internal/integration/TestRetryableWritesSpec
```

Comment:
Can timeout and return unknown results.

```
	=== RUN   TestRetryableWritesSpec/replaceOne.json/ReplaceOne_is_not_committed_on_first_attempt
	panic: test timed out after 5m0s
	running tests:
		TestRetryableWritesSpec (39s)
		TestRetryableWritesSpec/replaceOne.json (1s)
		TestRetryableWritesSpec/replaceOne.json/ReplaceOne_is_not_committed_on_first_attempt (0s)
```

Reply:
That should be in the comment in the file itself